packages/zoe/src/contracts/scaledPriceAuthority.js

```diff
@@ -58,8 +58,21 @@ export const start = async (
     transformSourceAmountOut: amountOut => floorDivideBy(amountOut, scaleOut),
   });
 
+  const withInitial = initialPrice
+    ? priceAuthority.then(
+        /** @param {PriceAuthority} pa */ pa =>
```

Comment:
this should be inferred from type of `priceAuthority`. Unfortunately what `makePriceAuthorityTransform` isn't quite a `PriceAuthority`:
> Type 'Promise<{ getQuoteIssuer: (brandIn: Brand<AssetKind>, brandOut: Brand<AssetKind>) => ERef<Issuer<"set">> | ERef<{ getBrand: () => ERef<...> | ERef<...>; ... 10 more ...; splitMany: (args_0: ERef<...>, args_1: Amount<...>[]) => ERef<...>; }>; ... 12 more ...; mutableQuoteWhenLT: (amountIn: Amount<...>, amountOutLimit...' is not assignable to type 'Promise<PriceAuthority>'.
  Type '{ getQuoteIssuer: (brandIn: Brand<AssetKind>, brandOut: Brand<AssetKind>) => ERef<Issuer<"set">> | ERef<{ getBrand: () => ERef<Brand<...>> | ERef<...>; ... 10 more ...; splitMany: (args_0: ERef<...>, args_1: Amount<...>[]) => ERef<...>; }>; ... 12 more ...; mutableQuoteWhenLT: (amountIn: Amount<...>, amountOutLimit:...' is not assignable to type 'PriceAuthority'.
    The types returned by 'getQuoteIssuer(...)' are incompatible between these types.
      Type 'ERef<Issuer<"set">> | ERef<{ getBrand: () => ERef<Brand<"set">> | ERef<{ isMyIssuer: (allegedIssuer: ERef<Issuer<AssetKind>>) => boolean | ... 2 more ... | PromiseLike<...>; getAllegedName: () => ERef<...>; getDisplayInfo: () => ERef<...>; getAmountShape: () => any; }>; ... 10 more ...; splitMany: (args_0: ERef<...>...' is not assignable to type 'ERef<Issuer<"set">>'.
        Type '{ getBrand: () => ERef<Brand<"set">> | ERef<{ isMyIssuer: (allegedIssuer: ERef<Issuer<AssetKind>>) => boolean | PromiseLike<boolean> | PromiseLike<...> | PromiseLike<...>; getAllegedName: () => ERef<...>; getDisplayInfo: () => ERef<...>; getAmountShape: () => any; }>; ... 10 more ...; splitMany: (args_0: ERef<...>, ...' is not assignable to type 'ERef<Issuer<"set">>'.
          Type '{ getBrand: () => ERef<Brand<"set">> | ERef<{ isMyIssuer: (allegedIssuer: ERef<Issuer<AssetKind>>) => boolean | PromiseLike<boolean> | PromiseLike<...> | PromiseLike<...>; getAllegedName: () => ERef<...>; getDisplayInfo: () => ERef<...>; getAmountShape: () => any; }>; ... 10 more ...; splitMany: (args_0: ERef<...>, ...' is not assignable to type 'Issuer<"set">'.
            The types returned by 'getBrand(...)' are incompatible between these types.
              Type 'ERef<Brand<"set">> | ERef<{ isMyIssuer: (allegedIssuer: ERef<Issuer<AssetKind>>) => boolean | PromiseLike<boolean> | PromiseLike<...> | PromiseLike<...>; getAllegedName: () => ERef<...>; getDisplayInfo: () => ERef<...>; getAmountShape: () => any; }>' is not assignable to type 'Brand<"set">'.
                Type 'PromiseLike<Brand<"set">>' is missing the following properties from type 'Brand<"set">': isMyIssuer, getAllegedName, getDisplayInfo, getAmountShape

I'll push a fix.